Dockerfile.frontend-base

```diff
@@ -5,9 +5,7 @@ RUN /root/bin/docker-bootstrap.sh frontend
 # Install other requirements
 RUN npm install -g jshint
 
-ADD src/api/Gemfile /obs/src/api/Gemfile
-ADD src/api/Gemfile.lock /obs/src/api/Gemfile.lock
-RUN chown -R frontend /obs/src/api
+ADD --chown=frontend:users src/api/Gemfile* /obs/src/api/
```

Comment:
Nitpick. Is `:users` needed? I would leave this line as:

`ADD --chown=frontend src/api/Gemfile* /obs/src/api/`